test/index.js

```diff
@@ -81,6 +82,8 @@ def ('$26')
     (a => b => c => d => e => f => g => h => i => j => k => l => m => n => o => p => q => r => s => t => u => v => w => x => y => z =>
        [a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p, q, r, s, t, u, v, w, x, y, z]);
 
+const suite = (_, run) => run ();
```

Comment:
This is a small hack to keep the tests running with minimal changes. I have noticed that this indirection makes the locations of test failures reported by Oletus less helpful; I am willing to tolerate this.

Reply:
See also https://github.com/bearror/oletus/issues/21